frontend/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,25 @@
+import "./Footer.css";
+
+function Footer() {
+
+    return (
+        <footer>
+            <div id={"on-github"}>
+                <p>
+                    Click one of the logos to see code, project and issues on Github...
+                </p>
+                <a href="https://github.com/SchulzEnrico/ESourc-es-" target="_blank">
+                    <img title={"Code on Github"} src="../src/assets/github.png" className="logo" alt="View Code on Github" />
```

Comment:
Der alt Text soll das Bild beschreiben nicht die Funktion das umliegenden Links. So können sehbehinderte Nutzer verstehen was auf den Bildern dargestellt ist.

Eine weitere Anmerkung: du möchtest, dass dein Link in einem neuen Tab geöffnet wird. Da du allerdings die PWA im standalone modus(siehe site.webmanifest) angezeigt wird, könnte das dazu führen, dass der Standardbrowser vom Gerät geöffnet wird. Das könnte die Nutzer eventuell irritieren.
```suggestion
                </p>
                <a href="https://github.com/SchulzEnrico/ESourc-es-" target="_blank">
                    <img title={"Code on Github"} src="../src/assets/github.png" className="logo" alt="GitHub-Logo" />
```

Reply:
Den alt="Text" ändere ich ab. 

 Dass die Links in einem neuen Tab geöffnet werden, muss in diesem Fall so sein. Da diese alle nicht in iframes eingebunden werden können. Zumal ich glaube, dass das webmanifest eh nicht zur Anwendung kommt. 
Aber lass da gerne später nochmal drüber sprechen.